.github/workflows/publish.yml

```diff
@@ -0,0 +1,17 @@
+name: Build and publish python package
+
+on:
+  release:
+    types: [ published ]
+
+jobs:
+  publish-service-client-package:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Publish PyPi package
+        uses: code-specialist/pypi-poetry-publish@v1
```

Comment:
I guess this repo is not "dangerous" but this wouldn't pass the Scorecards security action. I also think you'd need to add the action to our exception list (but not sure about that).